### ISSUE TITLE
Получение разделов в древесном виде

### ISSUE THREAD

**Issue by simixmc** [onlinerby/onliner-b2b-api#184]
Возможно ли получать разделы в древесном виде с получением родительских и дочерних разделов?

**Comment by sidorovich** [comment]
список разделов у нас плоский. структуры разделов в каталоге нет.
навигация существует отдельно

**Comment by simixmc** [comment]
Возможно ли получить навигацию через API?

**Comment by sidorovich** [comment]
нет, так как не планировалось, что это будет необходимо магазинам.
вы для решения какой задачи хотите использовать эту информацию?

**Comment by simixmc** [comment]
Стоит задача связи каталога площадки с каталогом магазина

**Comment by sidorovich** [comment]
используйте API или файл со структурой каталога
в навигации гораздо более сложные сущности используются и навигация меняется очень оперативно
